medusa/server/web/home/add_shows.py

```diff
@@ -308,7 +310,7 @@ def popularShows(self):
 
         try:
             recommended_shows = ImdbPopular().fetch_popular_shows()
-        except (StandardError, requests.RequestException) as e:
+        except (RequestException, socket.gaierror, StandardError) as e:
```

Comment:
Hmm that's raised by pyNMA. What does that have to do with fetching the imdb shows? I think that requests already handles that. Reason why pyNMA raises that, is because they directly use httplib.

I Don't see a good reason to start handling that exception everywhere??